graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/ElasticsearchClient.java

```diff
@@ -178,4 +189,22 @@ public static RequestOptions withTimeout(RequestOptions requestOptions, Duration
                 .build();
 
     }
+
+    public Optional<ResponseError> parseResponseException(ElasticsearchException ex) {
+        if (ex.getCause() != null) {
+            final Throwable[] suppressed = ex.getCause().getSuppressed();
+            if (suppressed.length > 0) {
+                final Throwable realCause = suppressed[0];
+                if (realCause instanceof ResponseException) {
+                    try {
+                        final ResponseError err = objectMapper.readValue(((ResponseException) realCause).getResponse().getEntity().getContent(), ResponseError.class);
+                        return Optional.of(err);
+                    } catch (IOException ioe) {
+                        //LOG.warn("Failed to parse exception", e);
```

Comment:
If the exception can be dropped, I would at least write a comment hat suggests so.

Reply:
returned it back, no idea why I commented it out in the first place :see_no_evil: